src/host/host.test.ts

```diff
@@ -59,6 +60,18 @@ describe("host", () => {
             setter!.set!("some new value");
             expect(mockDevToolsWindow.localStorage).toBeUndefined();
         });
+
+        it("verifies that session storage cannot be modified", async () => {

```

Comment:
Probably would be better to test that session storage actually still works, rather than cannot be modified, something like: 
```
        it("emulates session storage", async () => {
            const host = await import("./host");
            host.initialize(mockIframe);

            expect(mockDevToolsWindow.sessionStorage).toBeDefined();

            const expectedKey = "key";
            const expectedVal = "value";
            (mockDevToolsWindow as any).sessionStorage[expectedKey] = expectedVal;
            expect(mockDevToolsWindow.sessionStorage[expectedKey]).toEqual(expectedVal);
        });
```